dms/models/dms_file.py

```diff
@@ -601,3 +601,29 @@ def _compute_locked(self):
                 )
             else:
                 record.update({"is_locked": False, "is_lock_editor": False})
+
+    def create_dms_file_from_attachments(self, attachment_ids=None, view_type="list"):
```

Comment:
Why this is call `create_...` if it's not creating anything? What is its purpose? And please don't put empty lines inside a method, that is less readable. If you need to put a "section", just add a comment line and you can give such section "a title".

Reply:
Yeah. That makes sense. I changed it to `get`

Reply:
Video and Audio work now :) and now DMS even supports more previewable audio types than Odoo core (mp3)